app/javascript/packages/captcha-submit-button/captcha-submit-button-element.ts

```diff
@@ -0,0 +1,70 @@
+class CaptchaSubmitButtonElement extends HTMLElement {
+  connectedCallback() {
+    this.button.addEventListener('click', (event) => this.handleButtonClick(event));
+  }
```

Comment:
just checking my understanding, we won't need to worry about a disconnected callback like in #7779, because if this element goes out of scope, its child button will go too, so that listener will clean itself up

Reply:
> just checking my understanding, we won't need to worry about a disconnected callback like in #7779, because if this element goes out of scope, its child button will go too, so that listener will clean itself up

It's probably good practice to always clean up in a `disconnectedCallback` whatever's set up in a `connectedCallback`, though I've tended to make the pragmatic choice that the lifecycle of these elements is pretty simple and to avoid the extra code / test cases. It's probably technically possible that the parent could become detached while the button continues to exist, which may result in some odd behavior, though we'd never expect that in practice. #7779 is slightly different in nature, since it leaves open a dangling timeout, and the Node process wasn't terminating gracefully because there were open timeouts. I don't think I'd expect the same to happen for an event listener, and I would expect that the DOM would automatically remove any event listeners associated with elements that have been removed from the page (such as during [the cleanup between each test case](https://github.com/18F/identity-idp/blob/fd14459a6af21cd419c2397e34dd37b1dda5d4e8/spec/javascripts/support/dom.js#L97-L99)).